submissions/sprint_a/app.py

```diff
@@ -0,0 +1,14 @@
+from . import app, jsonify
+
+
+@app.route('/')
+def generate_epithets():
+
```

Comment:
On functions this small, docstrings really aren't necessary, but a docstring or remove the newline.